thorlcr/activities/choosesets/thchoosesetsslave.cpp

```diff
@@ -308,7 +308,6 @@ class ChooseSetsPlusActivity : public CSlaveActivity, implements ILookAheadStopN
     virtual void getMetaInfo(ThorDataLinkMetaInfo &info) const override
     {
         initMetaInfo(info);
-        info.buffersInput = true;
```

Comment:
buffersInput and canBufferInput were used interchangeably, wherever buffersInput OR canBufferInput existed, it was to inform a decision re. canStall()